freud/data.py

```diff
@@ -68,9 +68,33 @@ def generate_system(self, num_replicas=1, scale=1, sigma_noise=0, seed=None):
 
         Note:
             Positions are generated in the order of the instance's
-            ``basis_positions``. The first :math:`N_{replica}`
-            positions come from the first basis position, the next
-            :math:`N_{replica}` the second, etc.
+            ``basis_positions``. The first :math:`N_{replica}` positions come
+            from the first basis position, the next :math:`N_{replica}` the
```

Comment:
Since you're updating this note, it might be helpful to say explicitly that the behavior is analogous to `np.repeat` rather than `np.tile`.